.github/workflows/ci_email.yml

```diff
@@ -0,0 +1,16 @@
+on: check_suite
+name: CI email
+jobs:
+  sendEmail:
+    name: Send email
+    runs-on: ubuntu-latest
+    steps:
+    - name: Send email
+      uses: docker://fertapric/elixir-ci-email:latest
```

Comment:
We need to document how to build this image or include a link to the repo that builds it.